src/styles/index.css

```diff
@@ -0,0 +1,248 @@
+body {
+  margin: 0;
+  font-family: Roboto, 'Times New Roman', serif;
+}
+
+.container {
+  --column-count: 1;
+
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  justify-content: center;
+  gap: 23px 24px;
+}
+
+@media (min-width: 488px) {
+  .container {
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    --column-count: 4;
+  }
+}
+
+/* ====== Header ====== */
+.header {
+  display: flex;
+  height: 60px;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  padding: 3px 0 0;
+}
+
+.menu {
+  font-size: 12px;
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  position: relative;
+  margin: 0 10px;
+  height: 60px;
+  align-items: center;
+  text-decoration: none;
+  color: #060b35;
+  display: flex;
+}
+
+.menu li:first-child a {
+  margin-left: 0;
+}
+
+.menu li:last-child a {
+  margin-right: 0;
+}
+
+.link:hover {
+  color: #00acdc;
+}
+
+.link--is-active {
+  color: #00acdc;
+}
+
+.link--is-active::after {
+  content: '';
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 2px;
+  transform-origin: bottom left;
+}
+
+.link:hover::after {
+  background-color: rgb(0, 172, 220);
+}
+
+.container_logo {
+  width: 40px;
+}
+
+.free {
+  flex-grow: 1;
+}
+
+/* ====== Product cards ====== */
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-left: 4px;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(../images/star-active.svg);
+}
+
+.card {
+  display: flex;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.card__info {
+  text-align: left;
+  padding: 0 16px;
+}
+
+.card__imac-img {
+  margin-top: 32px;
+  margin-bottom: 36px;
+  width: 160px;
+  height: 134px;
+  padding: 0 19px;
```

Comment:
The `.card__imac-img` class has both `margin` and `padding` properties that are controlling its size. It's generally a good practice to use either `margin` or `padding` to adjust the element's position or size but not both unless there's a specific reason for it. Verify if both properties are necessary here.